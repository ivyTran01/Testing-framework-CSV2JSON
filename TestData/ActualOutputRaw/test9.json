[
{"name":"Ivy  T.","field2":"","ice  cream  lover":true,"height":5.2},
{"name":"Tanzil","field2":25,"ice  cream  lover":false,"height":5.9},
{"name":"Danny  Yu","field2":"","ice  cream  lover":true,"height":5.11},
{"name":"Abdullah  Amjad","field2":23,"ice  cream  lover":true,"height":5.1},
{"name":"Jocelyn","field2":22,"ice  cream  lover":true,"height":5.2},
{"name":"Christ","field2":23,"ice  cream  lover":false,"height":5.1},
{"name":"Monica","field2":50,"ice  cream  lover":true,"height":5.41}
]
