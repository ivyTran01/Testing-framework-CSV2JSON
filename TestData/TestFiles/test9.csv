name,age,ice  cream  lover,height
Ivy  T.,"",true,5.2
Tanzil,25,false,5.9
Danny  Yu,21,true,5.11
Abdullah  Amjad,23,true,5.10
Jocelyn,22,true,5.2
Christ,23,false,5.10
