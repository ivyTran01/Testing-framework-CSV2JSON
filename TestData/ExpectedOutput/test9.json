[
 {
  "name":"Ivy  T.",
  "field2":null,
  "ice  cream  lover":true,
  "height":5.2
 },
 {
  "name":"Tanzil",
  "field2":25.0,
  "ice  cream  lover":false,
  "height":5.9
 },
 {
  "name":"Danny  Yu",
  "field2":null,
  "ice  cream  lover":true,
  "height":5.11
 },
 {
  "name":"Abdullah  Amjad",
  "field2":23.0,
  "ice  cream  lover":true,
  "height":5.1
 },
 {
  "name":"Jocelyn",
  "field2":22.0,
  "ice  cream  lover":true,
  "height":5.2
 },
 {
  "name":"Christ",
  "field2":23.0,
  "ice  cream  lover":false,
  "height":5.1
 }
]